src/ir/block_builder_test.cc

```diff
@@ -29,12 +29,16 @@ namespace {
 
 class BlockBuilderTest : public testing::Test {
  public:
-  BlockBuilderTest() {
-    context_.RegisterOperator(std::make_unique<Operator>("core.plus"));
-    context_.RegisterOperator(std::make_unique<Operator>("core.minus"));
-    context_.RegisterOperator(std::make_unique<Operator>("core.merge"));
-    context_.RegisterOperator(std::make_unique<Operator>("core.choose"));
-  }
+  BlockBuilderTest()
```

Comment:
Add test to https://github.com/google-research/raksha/blob/main/src/ir/ir_context_test.cc?

Reply:
Done.